<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>381</width>
    <height>340</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <pointsize>22</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>물품관리시스템</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTableWidget" name="tableWidget">
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="showGrid">
       <bool>true</bool>
      </property>
      <property name="columnCount">
       <number>3</number>
      </property>
      <column>
       <property name="text">
        <string>제품명</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>갯수</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>생산일</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>381</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>메뉴</string>
    </property>
    <addaction name="actionInput"/>
    <addaction name="actionView"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>보기</string>
    </property>
    <addaction name="actionSearch"/>
    <addaction name="actionModify"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionInput"/>
   <addaction name="actionView"/>
   <addaction name="actionSearch"/>
   <addaction name="actionModify"/>
  </widget>
  <action name="actionInput">
   <property name="icon">
    <iconset>
     <normaloff>if_-08_3057693.png</normaloff>if_-08_3057693.png</iconset>
   </property>
   <property name="text">
    <string>입력</string>
   </property>
  </action>
  <action name="actionView">
   <property name="icon">
    <iconset>
     <normaloff>if_-13_3057628.png</normaloff>if_-13_3057628.png</iconset>
   </property>
   <property name="text">
    <string>보기</string>
   </property>
  </action>
  <action name="actionSearch">
   <property name="icon">
    <iconset>
     <normaloff>if_-56_3057662.png</normaloff>if_-56_3057662.png</iconset>
   </property>
   <property name="text">
    <string>검색</string>
   </property>
  </action>
  <action name="actionModify">
   <property name="icon">
    <iconset>
     <normaloff>if_-54_3057663.png</normaloff>if_-54_3057663.png</iconset>
   </property>
   <property name="text">
    <string>수정</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
